other/someValueInArray.js

```diff
@@ -0,0 +1,33 @@
+// Mengecek apakah salah satu valuenya termasuk di sebuah array
+
+let isiDapurSatu = ["panci", "penggorengan", "garpu"];
+let isiDapurDua = ["garpu", "gelas", "sendok"];
+let isiDapurTiga = ["sodet", "cangkir"];
```

Comment:
```suggestion
const isiDapurSatu = ["panci", "penggorengan", "garpu"];
const isiDapurDua = ["garpu", "gelas", "sendok"];
const isiDapurTiga = ["sodet", "cangkir"];
```
karena berisi `array` dan nilainya tidak berubah, lebih baik menggunakan `const`.
sekaligus bisa memberikan contoh practice yang baik

Reply:
Oke besok pagi ku ubah, abis batre laptop ku